client/src/components/LocationAutocomplete.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import * as esriService from "../services/esri_service";
+import { TextField, Typography, Button } from "@material-ui/core";
+
+const LocationAutocomplete = props => {

```

Comment:
I've found over time I prefer to destructure my props right off the bat, here, so I always have a clear list of props coming in. We should also think about adding prop-types so we can start to solidify things that way too. You can define specific data types like a stakeholder with something like `stakeholder: PropTypes.shape({lat: PropTypes.number.isRequired, ...etc})`